daemon/logic/worklog.go

```diff
@@ -100,13 +107,30 @@ func (w *Worklog) Resolve(ctx context.Context, n *observer.Notifier,
 		return nil, nil
 	}
 
-	return w.handlers[item.Type()].resolve(ctx, n, orgID, item)
+	handler := w.handlers[item.Type()]
+	result, err := handler.resolve(ctx, n, orgID, item)
+
+	if err != nil {
+		result = &apitypes.WorklogResult{
```

Comment:
Ah I see, it was here before, you jsut cleaned it up a bit. Still a question though! Perhaps add a comment?